graph/graph_test.go

```diff
@@ -272,9 +272,9 @@ func same(a, b graph.Graph) bool {
 			return false
 		}
 	}
-	for _, u := range a.Nodes() {
-		aFromU := a.From(u.ID())
-		bFromU := b.From(u.ID())
+	for _, u := range graph.NodesOf(a.Nodes()) {
```

Comment:
Why use range and not iterator here?

Reply:
In a test this is simpler and the cost is not an issue.